### PR TITLE
ignore files when using composer

### PR DESCRIPTION
**Pull request by totoprayogo1916**
**Description**
Exclude some files when users run `create-project`

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPdocs
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Review by paulbalandan — CHANGES_REQUESTED** [review]
`builds` is needed to quickly switch between stable and unstable versions such as to test snapshot builds or test new unreleased features. Do not put it here.

File: admin/starter/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.gitattributes export-ignore
+
```
Comment (samsonasik) [inline admin/starter/.gitattributes:2]: no need multiple lines, single line is ok

File: admin/starter/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.gitattributes export-ignore
+.github/ export-ignore
```
Comment (samsonasik) [inline admin/starter/.gitattributes:2]: `/` prefix need to be added to ensure only ignore root dir/file, see https://github.com/codeigniter4/CodeIgniter4/blob/develop/admin/framework/.gitattributes

**Review by paulbalandan — APPROVED** [review]

**Review by samsonasik — APPROVED** [review]

**Comment by samsonasik** [comment]
@totoprayogo1916 thank you for the PR.
@paulbalandan thank you for the review.

**Pull request merged**